### ISSUE TITLE
[Issue]: IPv4 connectivity lost since 10.9.x

### ISSUE THREAD

**Issue by twdkeule** [jellyfin/jellyfin#11968]
### Please describe your bug

Since 10.9.x Jellyfin cannot be accessed from Android, LG WebOs or web clients. Also DLNA does not work anymore.
I can see that Jellyfin only listens on tcp6:
```
thomas@tiki /e/jellyfin> sudo netstat -a -p -t | rg -i jellyfin
tcp6       0      0 [::]:8096               [::]:*                  LISTEN      470166/jellyfin
```
And indeed, connecting via web client to the IPv6 address works: 
`tcp6       0      0 tiki:8096               fe80::ba23:9064:d:61408 ESTABLISHED 470166/jellyfin`

Changing `<EnableIPv6>` to false makes ipv4 work again:
```
thomas@tiki /e/jellyfin> sudo netstat -a -p -t | rg -i jellyfin
tcp        0      0 0.0.0.0:8096            0.0.0.0:*               LISTEN      471273/jellyfin
```

But ideally it should still work when selecting both IPv6 and IPv4.

### Reproduction Steps

Network.xml:
```
> cat network.xml
﻿<?xml version="1.0" encoding="utf-8"?>
<NetworkConfiguration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
  <BaseUrl />
  <EnableHttps>false</EnableHttps>
  <RequireHttps>false</RequireHttps>
  <CertificatePath />
  <CertificatePassword />
  <InternalHttpPort>8096</InternalHttpPort>
  <InternalHttpsPort>8920</InternalHttpsPort>
  <PublicHttpPort>8096</PublicHttpPort>
  <PublicHttpsPort>8920</PublicHttpsPort>
  <AutoDiscovery>true</AutoDiscovery>
  <EnableUPnP>false</EnableUPnP>
  <EnableIPv4>true</EnableIPv4>
  <EnableIPv6>true</EnableIPv6>
  <EnableRemoteAccess>false</EnableRemoteAccess>
  <LocalNetworkSubnets />
  <LocalNetworkAddresses />
  <KnownProxies />
  <IgnoreVirtualInterfaces>false</IgnoreVirtualInterfaces>
  <VirtualInterfaceNames>
    <string>veth</string>
  </VirtualInterfaceNames>
  <EnablePublishedServerUriByRequest>false</EnablePublishedServerUriByRequest>
  <PublishedServerUriBySubnet />
  <RemoteIPFilter />
  <IsRemoteIPFilterBlacklist>false</IsRemoteIPFilterBlacklist>
</NetworkConfiguration>
```

### Jellyfin Version

10.9.0

### Environment

```markdown
- OS: Ubuntu 22.04.4 LTS
- Linux Kernel:
- Virtualization: None
- Clients: Android, LG WebOs or web clients
- Browser: Chrome
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- GPU Model:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking: regular host, fixed IPv4 address
- Storage: local
```

### Jellyfin logs

```shell
[2024-06-05 08:26:53.017 +02:00] [INF] Jellyfin version: "10.9.4"
[2024-06-05 08:26:53.040 +02:00] [INF] Environment Variables: ["[JELLYFIN_ADDITIONAL_OPTS, ]", "[JELLYFIN_WEB_OPT, --webdir=/usr/share/jellyfin/web]", "[JELLYFIN_FFMPEG_OPT, --ffmpeg=/usr/lib/jellyfin-ffmpeg/ffmpeg]", "[JELLYFIN_ARGS, $JELLYFIN_WEB_OPT $JELLYFIN_FFMPEG_OPT $JELLYFIN_SERVICE_OPT $JELLYFIN_NOWEBAPP_OPT $JELLFIN_ADDITIONAL_OPTS --datadir $JELLYFIN_DATA_DIR --configdir $JELLYFIN_CONFIG_DIR --logdir $JELLYFIN_LOG_DIR --cachedir $JELLYFIN_CACHE_DIR]", "[JELLYFIN_LOG_DIR, /var/log/jellyfin]", "[JELLYFIN_CONFIG_DIR, /etc/jellyfin]", "[JELLYFIN_CACHE_DIR, /var/cache/jellyfin]", "[JELLYFIN_USER, jellyfin]", "[JELLYFIN_DATA_DIR, /var/lib/jellyfin]"]
[2024-06-05 08:26:53.042 +02:00] [INF] Arguments: ["/usr/lib/jellyfin/bin/jellyfin.dll", "--webdir=/usr/share/jellyfin/web", "--ffmpeg=/usr/lib/jellyfin-ffmpeg/ffmpeg"]
[2024-06-05 08:26:53.042 +02:00] [INF] Operating system: "Ubuntu 22.04.4 LTS"
[2024-06-05 08:26:53.042 +02:00] [INF] Architecture: X64
[2024-06-05 08:26:53.043 +02:00] [INF] 64-Bit Process: True
[2024-06-05 08:26:53.043 +02:00] [INF] User Interactive: True
[2024-06-05 08:26:53.043 +02:00] [INF] Processor count: 4
[2024-06-05 08:26:53.043 +02:00] [INF] Program data path: "/var/lib/jellyfin"
[2024-06-05 08:26:53.043 +02:00] [INF] Log directory path: "/var/log/jellyfin"
[2024-06-05 08:26:53.043 +02:00] [INF] Config directory path: "/etc/jellyfin"
[2024-06-05 08:26:53.043 +02:00] [INF] Cache path: "/var/cache/jellyfin"
[2024-06-05 08:26:53.043 +02:00] [INF] Web resources path: "/usr/share/jellyfin/web"
[2024-06-05 08:26:53.043 +02:00] [INF] Application directory: "/usr/lib/jellyfin/bin/"
[2024-06-05 08:26:53.155 +02:00] [INF] Setting cache path: "/var/cache/jellyfin"
[2024-06-05 08:26:53.180 +02:00] [INF] Loading assemblies
[2024-06-05 08:26:53.279 +02:00] [INF] Defined LAN subnets: ["::1/128", "fe80::/10", "fc00::/7", "127.0.0.1/8", "10.0.0.0/8", "172.16.0.0/12", "192.168.0.0/16"]
[2024-06-05 08:26:53.279 +02:00] [INF] Defined LAN exclusions: []
[2024-06-05 08:26:53.279 +02:00] [INF] Used LAN subnets: ["::1/128", "fe80::/10", "fc00::/7", "127.0.0.1/8", "10.0.0.0/8", "172.16.0.0/12", "192.168.0.0/16"]
[2024-06-05 08:26:53.280 +02:00] [INF] Filtered interface addresses: ["127.0.0.1", "192.168.1.2", "100.125.139.107", "172.17.0.1", "172.30.32.1", "::1", "fe80::dacb:8aff:fe9d:5e73%2", "fd7a:115c:a1e0:ab12:4843:cd96:627d:8b6b", "fe80::8cc0:b884:2737:ba73%3", "fe80::42:5bff:fe94:6e9e%4", "fe80::42:2dff:fe65:1898%5", "fe80::dcf2:88ff:fe00:ac0f%7", "fe80::6496:eaff:fef9:81ac%9", "fe80::b4ad:dff:fefe:b410%11", "fe80::e4af:9bff:fe68:b10d%13", "fe80::ac61:1eff:fe83:c3fd%15", "fe80::30d4:43ff:fef4:3de9%17", "fe80::9cb1:79ff:fe8a:3219%19"]
[2024-06-05 08:26:53.281 +02:00] [INF] Bind Addresses ["::"]
[2024-06-05 08:26:53.281 +02:00] [INF] Remote IP filter is "Allowlist"
[2024-06-05 08:26:53.281 +02:00] [INF] Filtered subnets: []
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by gnattu** [comment]
> <EnableRemoteAccess>false</EnableRemoteAccess>

Currently you need to enable this if you want to access via IPv4 when IPv6 is enabled.

**Comment by disgustipated** [comment]
Can you test the dlls in the zip i posted here https://github.com/jellyfin/jellyfin-plugin-dlna/issues/55 to see if it resolves your dlna issue(and maybe the other devices you have listed)? i have an lg webos tv and my changes fixed it for me.

**Comment by twdkeule** [comment]
Apparently dlna was moved to a plugin. Activating that plugin worked for me.

**Comment by felix920506** [comment]
Should be fixed by #12094
